### PR TITLE
Test rbovirt against both master and stable

### PR DESCRIPTION
**Pull request by lzap**
We have stabilized both branches, now we are able to test them both.

### CODE REVIEW

File: fb-install-ovirt.bats [hunk 1]
```diff
@@ -148,9 +148,27 @@ ENDPOINT
...
     skip "Not supported on this distro, install deps manually"
   fi
 }
+
+@test "perform integration tests against master" {
+  if ! tIsRedHatCompatible; then
+    skip "Not supported on this distro"
+  fi
+  pushd rbovirt
+  run rake spec --trace
```
Comment (ekohl) [inline fb-install-ovirt.bats]: Maybe an explicit git checkout here instead of assuming?

Comment (lzap) [inline fb-install-ovirt.bats]: > Maybe an explicit git checkout here instead of assuming?

Sure, done.

-- 
Later,
 Lukas #lzap Zapletal

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**